Library/UIScrollView+Extensions.swift

```diff
@@ -24,7 +24,14 @@ extension UIScrollView {
       delay: 0.0,
       options: change.options,
       animations: { [weak self] in
-        self?.contentInset.bottom = change.frame.height
+        switch change.notificationName {
```

Comment:
Same thing as above...wonder if we can test this?

Reply:
None of our UIKit extensions currently have tests...

Reply:
![](https://media0.giphy.com/media/fkPxDJ2bpZX7q/giphy.gif?cid=790b76115d30a9a54f395430598cfccb&rid=giphy.gif)

don't worry about it then 😎